packages/backend/src/apps/pair/actions/prompt-model/index.ts

```diff
@@ -0,0 +1,63 @@
+import { IRawAction } from '@plumber/types'
+
+import { ZodError } from 'zod'
+import { fromZodError } from 'zod-validation-error'
+
+import { VALIDATION_ERROR_SOLUTION } from '@/apps/postman/common/constants'
+import appConfig from '@/config/app'
+import StepError from '@/errors/step'
+
+import { requestSchema, responseSchema } from './schema'
+
+const action: IRawAction = {
+  name: 'Prompt Model',
+  key: 'promptModel',
+  description: 'Prompt AI model for an answer',
+  arguments: [
+    {
+      label: 'Prompt',
+      key: 'prompt',
+      type: 'string' as const,
+      required: true,
+      variables: true,
+    },
+  ],
+
+  async run($) {
+    try {
+      const payload = {
+        prompt: requestSchema.parse($.step.parameters).prompt,
+        model: 'gpt-3.5-turbo',
```

Comment:
Technically should put this in an env var or user input, but we leave that to later when we see the usage and see how we want to productionize